### PR TITLE
Minor bug fixes in acquisition module

### LINKED ISSUE #24: Pyogrio engine does not support passing CRS anymore
[Geopandas upgrade to v1.0.0](https://github.com/geopandas/geopandas/releases) now defaults to the Pyogrio engine (instead of fiona), which automatically saves the GeoDataFrame's CRS when saving the DataFrame and does not allow for additional crs parameter to be set. Hence, passing the CRS is not supported anymore (_cf. region.py_):
```python
geometry_df.to_file(
            geom_dir.joinpath(f"{config.country}.geojson"),
            driver="GeoJSON",
        )
```

### PR DESCRIPTION
**Pull request by jsreuss**
Closes #24, #25

### CODE REVIEW

File: eurocropsml/acquisition/region.py [hunk 1]
```diff
@@ -162,7 +162,6 @@ def add_nuts_regions(
         geometry_df.to_file(
             geom_dir.joinpath(f"{config.country}.geojson"),
             driver="GeoJSON",
-            crs=shapefile.crs,
```
Comment (jmaces) [inline eurocropsml/acquisition/region.py:165]: Is this removal backward compatible, or should we specify a lower bound version on our pyogrio requirement now?

Comment (jsreuss) [reply eurocropsml/acquisition/region.py:165]: Good point! I'll check whether it's backward compatible.

Comment (jsreuss) [reply eurocropsml/acquisition/region.py:165]: It should be backward compatible. However, I just realized that not the `pyogrio` update caused the issue, but the `geopandas` update. Sorry for the confusion!

When saving the GeoDataFrame, older `geopandas` versions default to the `fiona` version, whereas the latest update defaults to the `pyogrio` engine. `pyogrio` does not allow for a CRS to be set, it just obtains it directly from the GeoDataFrame. Whereas `fiona` does allow for a CRS parameter and only obtains it from the GeoDataFrame if no CRS is passed. So this should be backward compatible.

Comment (jmaces) [reply eurocropsml/acquisition/region.py:165]: okay, thanks for checking :+1:

**Review by jmaces — APPROVED** [review]

**Pull request merged**